vivisect/__init__.py

```diff
@@ -1359,7 +1359,7 @@ def makeOpcode(self, va, op=None, arch=envi.ARCH_DEFAULT):
                 # which *removes* the deref flag...
                 # If we're an xref to something real, rip out the deref flag, but if we're
                 # an xref to a big fat 0, fuggedaboutit
-                if ptrdest and self.analyzePointer(ptrdest[0]):
+                if ptrdest and self.analyzePointer(ptrdest[0])[0]:
```

Comment:
since analyzePointer now returns a tuple, we need to grab what we need (type of target)